contracts.go

```diff
@@ -224,3 +220,68 @@ func compileContract(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func reviseContract(cmd *cobra.Command, args []string) error {
+
+	// read the params file
+	params_b, err := ioutil.ReadFile("params.toml")
+	if err != nil {
+		return err
+	}
+
+	// read the contract template
+	template_b, err := ioutil.ReadFile("template.md")
+	if err != nil {
+		return err
+	}
+
+	// open history file (or create history file if it doesn't exist)
+	file, err := os.OpenFile("history.toml", os.O_RDWR|os.O_CREATE, 0600)
```

Comment:
I can generate in new, but we still do have a hash of the original template. That code is in tact. Did you want me to add that to the history file as the first hash?